### PR TITLE
Feat/hexa 345 usage sections simplification

### PR DESCRIPTION
**Pull request by cheikhgwane**
This PR intends to display workspace's database environment variables (name, username, host..) into database page. We've also replaced the usage section for database and connections by links to the corresponding documentation on the wiki.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Display env variables on database view
- Replace usage sections content by links to the wiki

## Screenshots / screencast
Database view
image

Connection view
image

### CODE REVIEW

File: src/pages/workspaces/[workspaceSlug]/databases/index.tsx [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
 import ChevronLinkColumn from "core/components/DataGrid/ChevronLinkColumn";
 import Link from "core/components/Link";
 import Page from "core/components/Page";
-import Tabs from "core/components/Tabs";
 import Title from "core/components/Title";
 import { createGetServerSideProps } from "core/helpers/page";
 import { NextPageWithLayout } from "core/helpers/types";
 import { useTranslation } from "next-i18next";
-import { useRouter } from "next/router";
+import { DatabaseVariablesSection } from "workspaces/features/DatabaseVariablesSection/DatabaseVariablesSection";
```
Comment (qgerome) [inline src/pages/workspaces/[workspaceSlug]/databases/index.tsx:12]:
```suggestion
import DatabaseVariablesSection from "workspaces/features/DatabaseVariablesSection";
```

File: src/workspaces/features/DatabaseVariablesSection/DatabaseVariablesSection.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import { LockClosedIcon } from "@heroicons/react/24/outline";
+import DataGrid, { BaseColumn } from "core/components/DataGrid";
+import { TextColumn } from "core/components/DataGrid/TextColumn";
+import { slugify } from "workspaces/helpers/connection";
+
+type DatabaseVariablesSectionProps = {
+  workspace: DatabaseVariablesSection_WorkspaceFragment;
+};
+
+export const DatabaseVariablesSection = (
```
Comment (qgerome) [inline src/workspaces/features/DatabaseVariablesSection/DatabaseVariablesSection.tsx:14]: No need to export it

File: src/workspaces/features/DatabaseVariablesSection/DatabaseVariablesSection.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        )}
+      </BaseColumn>
+      <BaseColumn
+        className="flex justify-start gap-x-2 text-gray-900"
+        label={t("Value")}
+      >
+        {(field) => (
+          <>
+            {field.secret && <LockClosedIcon className="h-3 w-3" />}
+            {field.secret && "*********"}
```
Comment (qgerome) [inline src/workspaces/features/DatabaseVariablesSection/DatabaseVariablesSection.tsx:75]:
```suggestion
            {field.secret && field.value && "*********"}
```

**Review by qgerome — CHANGES_REQUESTED** [review]

**Review by qgerome — CHANGES_REQUESTED** [review]
In a workspace where I'm a admin, I don't see the '******' for the password. You also need to add a "display" button next to it to display the password if the user has the right.

**Comment by cheikhgwane** [comment]
> In a workspace where I'm a admin, I don't see the '******' for the password. You also need to add a "display" button next to it to display the password if the user has the right.

It's weird because I 've tested with an admin but it was displayed. It doesn't show  with the non-admin. I'll add the display button.

**Review by pvanliefland — CHANGES_REQUESTED** [review]
Still to do :

- [x] Adapt the PR after the backend changes
- [x] Replace "variables" by "connection parameters"
- [x] For connections, replace "Usage examples" by usage
- [x] Make the connection view similar to the database view (no separator between fields and usage)

**Pull request merged**